### ISSUE TITLE
[469b] Crash on startup when using OpenGL

### ISSUE THREAD

**Issue by Mantha065** [OldUnreal/UnrealTournamentPatches#792]
![error when running OpenGL](https://user-images.githubusercontent.com/55818816/163508939-0b01f813-3191-4314-9d88-e8a54093b584.png)

LOG:
[UnrealTournament.log](https://github.com/OldUnreal/UnrealTournamentPatches/files/8493485/UnrealTournament.log)

**Comment by stijn-volckaert** [comment]
Are you using the OpenGLDrv.dll that came with the patch? And could you upload your UnrealTournament.log immediately after the game crashes? The log you attached to the bug report shows you're using XOpenGLDrv and it has no crash info.

**Comment by Mantha065** [comment]
That was the log I generated immediately after the crash, I believe. I got a n
[UnrealTournament.log](https://github.com/OldUnreal/UnrealTournamentPatches/files/8498760/UnrealTournament.log)
ew one though, just in case. To be clear, I started the game and immediately got the "UT99 closed unexpectedly" thing from the last time it had crashed. I clicked "Change 3D video device", clicked OpenGL, and then clicked Run. The game of course immediately crashed, and then I went and got the log.

As for using the OpenGLDrv.dll that came with the patch, I believe I'm using that. To install 469b, I downloaded the .exe and installed it that way. Looking at the OpenGLDrv.dll file in the System folder, it says that file was last modified Dec 25 2002...I assume that's not the correct version. How do I acquire the correct OpenGLDrv.dll?
